shapedetector.py

```diff
@@ -66,7 +84,14 @@ def detect_shapes():
         M = cv2.moments(c)
 
         # ignore contours that are too small to be species
-        if c.shape[0] > 2 and area/(resized.shape[0]*resized.shape[1]) > 0.002:
+        UPPERCONTOURTHRESH = 0.25
+        LOWERCONTOURTHRESH = 0.002
+
+        if c.shape[0] > 2 and\
+        area/(resized.shape[0]*resized.shape[1]) > LOWERCONTOURTHRESH and\
```

Comment:
you should align the other parts of the if clause with the first one, ie this line should be

```suggestion
           area/(resized.shape[0]*resized.shape[1]) > LOWERCONTOURTHRESH and\
```